### ISSUE TITLE
Request: spontaneous metamagic

### ISSUE THREAD

**Issue by Truinto** [Truinto/DarkCodex#63]
### Discussed in https://github.com/Truinto/DarkCodex/discussions/61

<div type='discussions-op-text'>

<sup>Originally posted by **syntaxdot** June  4, 2022</sup>
Hey, Truinto! 
Thanks a lot for your work! 
Could you please add a feat that would allow spontaneous casters to apply metamagic to spells and keep the standard casting time?</div>

**Comment by RevanStarforge** [comment]
not the author, so whatever he would grant this request or not, i have another recommendation to you.

to my knowledge there is no such feat in the game... creating would be akin to cheating, and if you're fine with cheating however, you can just use toybox and give yourself Arcane Apotheosis ability. it's the capstone power of Arcane Bloodline (sorcerer), and it does exactly what you want. I give it to myself time to time

**Comment by Truinto** [comment]
The blueprint for that feature reads completely different from the source material. Apparently it grants 100 bonus to concentration instead affecting metamagic. Are you sure you have used that before? Perhaps you at using a mod that restores the feature?

**Comment by RevanStarforge** [comment]
holy shit you're right, it worked like that in kingmaker, and because most bluepritings were identical i assumed they kept it,

come to think of it, i havent played a sorcerer yet in wrath, could you check other arcane bloodline arcana ability if it mentions something like that? perhaps they moved into arcana itself with level dependency... if so, that means this dude's requests is much more important all of a sudden ^^

**Comment by RevanStarforge** [comment]
what did you add if i might ask exactly? a feat like the OP requested, or did you made arcane apotheosis work as it suppose to?

**Comment by RevanStarforge** [comment]
> I checked the code and I cannot find any logic that would allow to skip the spontaneous casting time increase. I already wrote a patch, it's pretty simple.

ok i just saw, that you added to arcane apotheosis...

i checked the srd to be sure and apperently there is this ability in arcane sorcerer bloodline;


Metamagic Adept ([Ex]): At 3rd level, you can apply any one metamagic feat you know to a spell you are about to cast without increasing the casting time. You must still expend a higher-level spell slot to cast this spell. You can use this ability once per day at 3rd level and one additional time per day for every four sorcerer levels you possess beyond 3rd, up to five times per day at 19th level. At 20th level, this ability is replaced by arcane apotheosis.


And arcane apotheosis reads;
Arcane Apotheosis ([Ex]): At 20th level, your body surges with arcane power. You can add any [metamagic feats](https://www.d20pfsrd.com/feats#TOC-Metamagic-Feats) that you know to your spells without increasing their casting time, although you must still expend higher-level spell slots. Whenever you use magic items that require charges, you can instead expend spell slots to power the item. For every three levels of spell slots that you expend, you consume one less charge when using a magic item that expends charges.



basically  making metamagic adept infinite and always on...


is there anything referencing to the metamagic adept in the blue prints? if not, can you add that too?
also, i'll remind because i think there are seperate abilities (if not or if you remembered already, i apologize) but reminder: sage bloodline and eldritch scion also will need these changes

**Comment by Truinto** [comment]
I was in a rush, because I forgot that SpellPouch shares dependency. I haven't actually tested that feature. Give me an extra day.

And metamagic adept gives various bonuses to concentration instead of RAW.

![1](https://user-images.githubusercontent.com/47917181/172472507-a90ed419-45af-420e-979e-60711e1c0b0f.png)

Will need to see how to fix that.

**Comment by Truinto** [comment]
I managed to implement the original feature, minus the item charge thing. Which I may also tackle.

**Comment by Truinto** [comment]
Eh, maybe later. I consider this request fulfilled.
